.ado/templates/build-rnw.yml

```diff
@@ -53,12 +55,15 @@ steps:
         /p:PreferredToolArchitecture=${{parameters.preferredToolArchitecture}}
         /p:PlatformToolset=${{parameters.platformToolset}}
         /p:BaseIntDir=$(BaseIntDir)
+        /bl:${{ parameters.BuildLogDirectory }}\MsBuild.binlog
+        /flp1:errorsonly;logfile=${{ parameters.BuildLogDirectory }}\MsBuild.err
+        /flp2:warningsonly;logfile=${{ parameters.BuildLogDirectory }}\MsBuild.wrn
+        /flp3:verbosity=diagnostic;logfile=${{ parameters.BuildLogDirectory }}\MsBuild.log
```

Comment:
wouldn't different builds end up in the same directory with the same name and therefore overwrite each other?

Reply:
A build only runs once on a machine in a job in our CI. And when uploading, azure devops does not support merging multiple artifacts, so the build logs artifact  is uniquely named for the job.